packages/collapsible/rc/index.js

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useState, useEffect } from 'react'
```

Comment:
I think if we accept this hook it should probably be moved to either utils or hooks specific package or should we start creating hooks packages?

Reply:
since it's part of Collapsible it's probably fine for now. if it was more generic(like useToggle) it say we put it with utils or core maybe

Reply:
I was thinking if we adopt the new drawer then the collapsible component could either stay as is but this new hook which renders no markup might need to co somewhere else. We actually could use it to update vertical tabs.